GeorgianMemenator/Scenes/DogBreedsList/DogBreedsListViewModel.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  DogBreedsListViewModel.swift
+//  GeorgianMemenator
+//
+//  Created by Qeti Avanashvili on 08.01.23.
+//  Copyright © 2023 Shalva Avanashvili. All rights reserved.
+//
+
+import Foundation
```

Comment:
Lets create the empty class for `DogBreedsListViewModel`, please check how `RedditViewModel` is created.